cmd/build/helmify/static/README.md

```diff
@@ -73,23 +73,27 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | emitAuditEvents                              | Emit K8s events in gatekeeper namespace for audit violations (alpha feature)           | `false`                                                                   |
 | logDenies                                    | Log detailed info on each deny                                                         | `false`                                                                   |
 | logLevel                                     | Minimum log level                                                                      | `INFO`                                                                    |
 | image.pullPolicy                             | The image pull policy                                                                  | `IfNotPresent`                                                            |
 | image.repository                             | Image repository                                                                       | `openpolicyagent/gatekeeper`                                              |
-| image.release                                | The image release tag to use                                                           | Current release version: `v3.5.0-rc.1`                                         |
+| image.release                                | The image release tag to use                                                           | Current release version: `v3.5.0-rc.1`                                    |
 | image.pullSecrets                            | Specify an array of imagePullSecrets                                                   | `[]`                                                                      |
 | resources                                    | The resource request/limits for the container image                                    | limits: 1 CPU, 512Mi, requests: 100mCPU, 256Mi                            |
 | nodeSelector                                 | The node selector to use for pod scheduling                                            | `kubernetes.io/os: linux`                                                 |
 | affinity                                     | The node affinity to use for pod scheduling                                            | `{}`                                                                      |
 | tolerations                                  | The tolerations to use for pod scheduling                                              | `[]`                                                                      |
+| certificates.generate                        | Generates a secret named `gatekeeper-webhook-server-cert` with CA and certificate      | `true`                                                                    |
```

Comment:
Maybe something like "When `true`, certificate generation and rotation is handled by Gatekeeper. Disable to support third-party certificate management"

Reply:
👍